### ISSUE TITLE
Cannot install Sober

### ISSUE THREAD

**Issue by MarkInCrack** [vinegarhq/sober#262]
### Acknowledgement of checking existent issues

- [X] I have checked if my issue has already been reported, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

Debian 10 Linux
Huayra 5.0 (austral)
Terminal:
estudiante@juanamanso:~$ flatpak install --user https://sober.vinegarhq.org/sober.flatpakref
error: Unable to load summary from remote sober-origin: GPG verification enabled, but no summary found (check that the configured URL in remote config is correct)

And i tried downloading it from the "sober.flatpakref" file and it gave me the same error

**Comment by m-attz** [comment]
Make sure that your URL is clean (no whitespaces, etc). This usually happens when the URL is wrong (despite it not looking incorrect).

**Comment by MarkInCrack** [comment]
> Make sure that your URL is clean (no whitespaces, etc). This usually happens when the URL is wrong (despite it not looking incorrect).

Now this appears
estudiante@juanamanso:~$ flatpak install --user https://sober.vinegarhq.org/sober.flatpakref
error: The application org.vinegarhq.Sober/x86_64/master requires the runtime org.gnome.Platform/x86_64/47 which was not found

**Comment by m-attz** [comment]
Try this:
1. flatpak repair --user
2. flatpak remove --unused
3. flatpak update

then attempt to install it again.

**Comment by MarkInCrack** [comment]
> Try this:
> 
>     1. flatpak repair --user
> 
>     2. flatpak remove --unused
> 
>     3. flatpak update
> 
> 
> then attempt to install it again.

This happened:

estudiante@juanamanso:~$ flatpak repair --user
Working on the user installation at /home/estudiante/.local/share/flatpak
Verifying newcp-origin:app/io.atom.electron.NewCP/x86_64/stable…
Verifying deploy/app/io.atom.electron.NewCP/x86_64/stable…
Verifying sober-origin:appstream2/x86_64…
Remote newcp-origin for ref app/io.atom.electron.NewCP/x86_64/stable is disabled
Pruning objects
estudiante@juanamanso:~$ flatpak remove --unused


        ID                                            Arch       Branch
 1. [-] org.freedesktop.Platform.GL.default           x86_64     22.08
 2. [-] org.freedesktop.Platform.GL.default           x86_64     22.08-extra
 3. [-] org.freedesktop.Platform.VAAPI.Intel          x86_64     1.6
 4. [-] org.freedesktop.Platform.VAAPI.Intel          x86_64     22.08
 5. [-] org.freedesktop.Platform.ffmpeg               x86_64     1.6
 6. [-] org.freedesktop.Platform                      x86_64     22.08
 7. [-] org.freedesktop.Platform.GL.Debug.default     x86_64     22.08
 8. [-] org.freedesktop.Platform.GL.Debug.default     x86_64     22.08-extra
 9. [-] org.freedesktop.Platform.Locale               x86_64     22.08
10. [-] org.freedesktop.Platform.openh264             x86_64     2.2.0
11. [-] org.gnome.Platform                            x86_64     3.26
12. [-] org.gnome.Platform.Locale                     x86_64     3.26

Uninstall complete.
estudiante@juanamanso:~$ flatpak update
Looking for updates…
Nothing to do.
estudiante@juanamanso:~$ flatpak install --user https://sober.vinegarhq.org/sober.flatpakref
error: The application org.vinegarhq.Sober/x86_64/master requires the runtime org.gnome.Platform/x86_64/47 which was not found

**Comment by kirbix12** [comment]
Run `flatpak install org.gnome.Platform`

Then enter `2` then `14`

**Comment by MarkInCrack** [comment]
$ flatpak install --user https://sober.vinegarhq.org/sober.flatpakref

org.vinegarhq.Sober permissions:
    network        fallback-x11       pulseaudio       wayland
    x11            devices            dri              file access [1]

    [1] xdg-run/app/com.discordapp.Discord:create, xdg-run/discord-ipc-0


        ID                    Arch     Branch   Remote         Descarga
 1. [✓] org.vinegarhq.Sober   x86_64   master   sober-origin   2,0 MB / 2,0 MB

Warning: Not exporting file org.vinegarhq.Sober.metainfo.xml of unsupported type.
Installation complete.

**Comment by MarkInCrack** [comment]
Now when i try to open Sober from the menu it doesn't run, i clicked something that said something about Google Play and then it said "Installing Sober" then i pressed a button and the app closed
![help](https://github.com/user-attachments/assets/d003cfec-95aa-4afd-84c4-3d1803d1ca2b)
Now i can't open sober :(

**Comment by MarkInCrack** [comment]
It opened only when i just installed it, after that it won't work

**Comment by m-attz** [comment]
Can you make sure that the process is not running (in the case that you are still experiencing this issue)?

1. flatpak ps

**Comment by MarkInCrack** [comment]
> Can you make sure that the process is not running (in the case that you are still experiencing this issue)?
> 
>     1. flatpak ps

It shows
$ flatpak ps
Instance   PID  Application         Runtime
2717529310 6311 org.vinegarhq.Sober org.gnome.Platform
But the app doesn't open

**Comment by m-attz** [comment]
Try running this command first then attempt to run sober:
1. flatpak kill org.vinegarhq.Sober

**Comment by MarkInCrack** [comment]
> Can you make sure that the process is not running (in the case that you are still experiencing this issue)?
> 
>     1. flatpak ps

$ flatpak ps
Instance   PID  Application         Runtime
2717529310 6311 org.vinegarhq.Sober org.gnome.Platform
But the app doesn't open

**Comment by MarkInCrack** [comment]
> Try running this command first then attempt to run sober:
> 
>     1. flatpak kill org.vinegarhq.Sober

I tried but it didn't open, when i tried "flatpak ps" it showed nothing

**Comment by m-attz** [comment]
That's really strange. Try running it from the command line: 
1. flatpak run org.vinegarhq.Sober

**Comment by MarkInCrack** [comment]
It didn't open, and from some testing i realized that it just runs for a few seconds, without opening the program

$ flatpak ps
Instance PID  Application         Runtime
14345227 4814 org.vinegarhq.Sober org.gnome.Platform
estudiante@juanamanso:~$ flatpak ps
Instance PID  Application         Runtime
14345227 4814 org.vinegarhq.Sober org.gnome.Platform
estudiante@juanamanso:~$ flatpak ps
Instance PID  Application         Runtime
14345227 4814 org.vinegarhq.Sober org.gnome.Platform
estudiante@juanamanso:~$ flatpak ps
estudiante@juanamanso:~$ 

The Instance only showed for 3 seconds

**Comment by MarkInCrack** [comment]
I updated Sober, it opened to update, but after i "open roblox", it doesn't run, i haven't installed a roblox apk, and running the run command, it won't open

**Comment by reflexran** [comment]
Can you run `sudo dmesg | grep sober`

**Comment by MarkInCrack** [comment]
user:~$ sudo dmesg | grep sober
[sudo] password for user:           
[58976.071921] sober[21591]: segfault at ffffffffffbffffe ip 00007fd722ec2ecd sp 00007ffc17470dd8 error 5 in libc.so.6[7fd722e37000+175000]
[59323.320252] sober[22187]: segfault at ffffffffffbffffe ip 00007fd15b8c7ecd sp 00007ffe05dbcc58 error 5 in libc.so.6[7fd15b83c000+175000]
[59330.461521] sober[22298]: segfault at ffffffffffbffffe ip 00007fb93c708ecd sp 00007ffc6b43b408 error 5 in libc.so.6[7fb93c67d000+175000]
[59905.730537] sober[22965]: segfault at ffffffffffbffffe ip 00007f035ff70ecd sp 00007fffbfc17638 error 5 in libc.so.6[7f035fee5000+175000]

**Comment by MarkInCrack** [comment]
How can i fix this?

**Comment by reflexran** [comment]
Do you know the CPU of this laptop? judging from the os it seems like this is a government provided laptop.

Run `sudo apt install neofetch` then `neofetch` and post the output here if you don't how the CPU.

**Comment by reflexran** [comment]
You can also just run `grep sse /proc/cpuinfo` and post the output here

**Comment by MarkInCrack** [comment]
Yes, it's the PC of the government, but it's the best i have, from Argentina
~$neofetch
...
CPU: Intel Celeron N4020 (2) @ 2.800GH

**Comment by m-attz** [comment]
Do you have any firewall inbound/outbound rules in place or a DNS that may not be resolving any requests?

**Comment by tunis4** [comment]
your kernel is too outdated to run sober, you need to update your distro

**Comment by MarkInCrack** [comment]
> Do you have any firewall inbound/outbound rules in place or a DNS that may not be resolving any requests?
How do i check if i have a firewall?
